plugins/chat/app/serializers/base_chat_channel_membership_serializer.rb

```diff
@@ -8,5 +8,10 @@ class BaseChatChannelMembershipSerializer < ApplicationSerializer
              :chat_channel_id,
              :last_read_message_id,
              :unread_count,
-             :unread_mentions
+             :unread_mentions,
+             :meta
+
+  def meta
```

Comment:
See other comment, this should be on `ChatChannelSerializer` instead, since a user may not have a membership for the channel yet.